content/en/synthetics/api_tests/_index.md

```diff
@@ -61,11 +61,11 @@ Define the request you want to be executed by Datadog:
 
 {{% tab "SSL Test" %}}
 
-{{< img src="synthetics/api_tests/make-ssl-request.png" alt="Make SSL Request"  style="width:80%;" >}}
+{{< img src="synthetics/api_tests/make-ssl-request.png" alt="Define SSL Request"  style="width:80%;" >}}
 
 1. **Choose request type**: `SSL`
 2. Specify the `Host` and the SSL `Port`. By default, the port is set to _443_.
-3. **Name**: The name of your API test.
+3. **Name**: The name of your SSL test.
 4. **Select your tags**: The tags attached to your SSL test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>` on the [Synthetic Monitoring page][1].
 5. **Locations**: The Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run a Synthetic API test on a private endpoint not accessible from the public internet.
```

Comment:
```suggestion
5. **Locations**: The Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run your Synthetic SSL test on a private host not accessible from the public internet.
```

Reply:
And I cannot seem to suggest on the HTTP test section but we probably want to do the same change on that one as well